MovieMVP/MovieMVP/MovieDetailTableViewModule/Presenter/MovieDetailsPresenter.swift

```diff
@@ -0,0 +1,50 @@
+// MovieDetailsPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Презентер экрана "Подробнее о фильме"
+final class MovieDetailsPresenter: MovieDetailsTableViewPresenterProtocol {
+    // MARK: - Public properties
+
+    var movieId: Int?
+    var movieDetail: MovieDetail?
+
+    // MARK: - Private properties
+
+    private let networkService: NetworkServiceProtocol
+    private var view: MovieDetailsTableViewProtocol?
+    private var router: RouterProtocol?
+
+    // MARK: - Init
+
+    init(
+        view: MovieDetailsTableViewProtocol,
+        networkService: NetworkServiceProtocol,
+        movieId: Int?,
+        router: RouterProtocol?
+    ) {
+        self.view = view
+        self.networkService = networkService
+        self.movieId = movieId
+        self.router = router
+        setMovieDetail(id: movieId)
+    }
+
+    // MARK: - Public methods
+
+    public func setMovieDetail(id: Int?) {
+        networkService.fetchMovieDetails(id: id ?? 0) { [weak self] result in
+            guard let self = self else { return }
+            DispatchQueue.main.async {
+                switch result {
+                case let .success(json):
+                    self.movieDetail = MovieDetail(json: json)
```

Comment:
```suggestion
                    self.movieDetail = MovieDetail(json: json)
```
JSON парусим на уровень ниже в апи сервисе